src/NumField/Subfields.jl

```diff
@@ -29,7 +29,7 @@ function _subfield_basis(K::S, as::Vector{T}) where {
       (Fvs,phivs) = sub(Kvs, typeof(Fvs)[Fvs, abvs])
       if dF != length(Fvs.gens) # dim(Fvs)
         dF = length(Fvs.gens) # dim(Fvs)
-        append!(nbs, abs)
+        ppend!(nbs, abs)
```

Comment:
ppend? for m it is truncated. The "a" is missing

Reply:
oops